getCalspec/rebuild.py

```diff
@@ -43,9 +45,24 @@ def add_astroquery_id(df):
 
 
 def add_alt_star_name(df):
+    name_columns = [name for name in df.columns if "name" in name.lower()]
+    print(name_columns)
```

Comment:
I'm not sure you want this `print` statement left in now you're done with development. If you _do_ want to leave it in I'd recommend putting some context with it, like `f"Columns to be used for identifying stars: {name_columns}"` or something like that.